### DISCUSSION TITLE
Fortio "2.0"

### DISCUSSION THREAD

**Discussion by ldemailly** [fortio/fortio#558] [Ideas]
Considering bumping the version to 2.0 in order to clean up a bunch of legacy options and change to better defaults instead of backward compatible defaults

Could also combine this with some bigger changes, maybe UI revamp too etc

Thoughts?

**Comment by ktalg** [comment]
How about supporting multiple application layer protocols? Redis, MySQL, Mongo, etc., and generate corresponding data graphs for each protocol test. There seems to be no such tool at present. It would be great to provide such support!

In addition, I think you could consider adding more **"dynamic"** functionality to Fortio, such as allowing the user to enter some kind of rule to dynamically generate the header and body for each request on the client side;The server can return a different response or perform a different action for each request

**Reply by ktalg** [reply]
Another suggestion is to provide the ability to visualize the test data in real time, such as viewing the current response latency per second on a graph,like https://github.com/tsenart/vegeta#usage-real-time-analysis

**Reply by ldemailly** [reply]
So for the realtime part you can do that now with `fortiotel`
